includes/polling/os/allied.inc.php

```diff
@@ -1,9 +1,18 @@
 <?php
 
-//Configuration file for Allied Telesis/Telesyn products NOT running Alliedware Plus 5.x.x.
-//See awplus.inc.php for Alliedware Plus 5.x.x OS configuration. 
-
+/*
```

Comment:
Don't follow the old code, we don't need all of these comments. The data it's referencing should be made available in the snmprec files for testing anyway

Reply:
So are you essentially saying get rid of pretty much all the comments e.g.

```

+/*
+Configuration file for Allied Telesis/Telesyn products NOT running Alliedware Plus 5.x.x.
 +See awplus.inc.php for Alliedware Plus 5.x.x OS configuration.
 + Configuration file covers:
 +- Alliedware
 +- Alliedware Plus v2.x.x.x
 +- AT-S39/41 circa 2003
 +- WebSmart OS
 +Note: ATI AT-8000S and AT-8000GS run RADLAN OS. 
 +*/ 
  
?